cpp/tests/interop/from_arrow_host_test.cpp

```diff
@@ -215,6 +221,244 @@ TYPED_TEST(FromArrowHostDeviceTestDurationsTest, DurationTable)
   CUDF_TEST_EXPECT_TABLES_EQUAL(got_cudf_table->view(), from_struct);
 }
 
+template <typename T>
+using fp_wrapper = cudf::test::fixed_point_column_wrapper<T>;
+
+TYPED_TEST(FromArrowHostDeviceTestDecimalsTest, FixedPointTable)
+{
+  using T = TypeParam;
+  using namespace numeric;
+
+  auto const precision = []() {
+    if constexpr (std::is_same_v<T, int64_t>)
+      return 18;
+    else
+      return cudf::detail::max_precision<T>();
+  }();
```

Comment:
Use the `get_decimal_precision()` utility from `array_utils.hpp`?
https://github.com/rapidsai/cudf/pull/17422/files#diff-05280325f14b4019fc93c51fdd88ac57f5d58183aa008e619f853b117e0c2901R412

Reply:
done

Reply:
There are 4 more of these
https://github.com/rapidsai/cudf/pull/17422/files#diff-dba75f5fe9619a48cd45b04cbe3d568512a8df46cc228cb10e4ac465157a2ca8R458
https://github.com/rapidsai/cudf/pull/17422/files#diff-dba75f5fe9619a48cd45b04cbe3d568512a8df46cc228cb10e4ac465157a2ca8R491
https://github.com/rapidsai/cudf/pull/17422/files#diff-dba75f5fe9619a48cd45b04cbe3d568512a8df46cc228cb10e4ac465157a2ca8R527
https://github.com/rapidsai/cudf/pull/17422/files#diff-dba75f5fe9619a48cd45b04cbe3d568512a8df46cc228cb10e4ac465157a2ca8R563

Reply:
bah, missed them cuz they were in another file. Fixed those spots too!